### PR TITLE
Bashbashbash patch 1

### PR DESCRIPTION
**Pull request by bashbashbash**
Updated for issue #2

### CODE REVIEW

**Review by kastolars — CHANGES_REQUESTED** [review]
I have added a checklist to the issue to show what is only needed.

**Comment by bashbashbash** [comment]
updated

File: README.md [hunk 1]
```diff
@@ -1 +1,57 @@
...
+
+## Installation
+Please select the branch that reflects your OS.
+[Linux Binary](https://github.com/SIGBlockchain/aurum_client/tree/linux_release)
+[Mac OS X Binary](https://github.com/SIGBlockchain/aurum_client/tree/mac_release)
+[Windows Binary](https://github.com/SIGBlockchain/aurum_client/tree/windows_release)
+Download this branch into your local environent. 
+Open a console or terminal window and navigate to the the aurum_client directory
+Type the following:
+sudo chmod +x setup.sh
```
Comment (kastolars) [inline README.md]: I'm not sure these particular instructions will work for all OS's, perhaps specify that?

File: README.md [hunk 2]
```diff
@@ -1 +1,57 @@
...
+**ascii logo**
+Welcome! Let's get everything set up.
+can't load package: package .: no Go files in /Users/zlabaschin/.go/src/github.com/SIGBlockchain/aurum_client
+Initializing Aurum wallet...
+Wallet setup complete.
+Okay your wallet's ready.
+Run ./client -h or ./client --help to see all commands
+Check out the wiki on the github page for more info: https://github.com/SIGBlockchain/aurum_client/wiki
+
+A file called aurum_wallet.json was created. Copy this file and store it somewhere secure. __It contains your private key. If you lose you private key, you will lose access to your wallet and it is not recoverable.__
```
Comment (kastolars) [inline README.md]: Urge the user to save a copy of `aurum_wallet.json` to another secure location just in case.

File: README.md [hunk 3]
```diff
@@ -1 +1,57 @@
-# aurum_client
\ No newline at end of file
+<img src = "assets/aurum_logo_readme.jpg"  alt="drawing" width="200"/>
+
+## Installation
+Please select the branch that reflects your OS.
+[Linux Binary](https://github.com/SIGBlockchain/aurum_client/tree/linux_release)
+[Mac OS X Binary](https://github.com/SIGBlockchain/aurum_client/tree/mac_release)
+[Windows Binary](https://github.com/SIGBlockchain/aurum_client/tree/windows_release)
+Download this branch into your local environent.
```
Comment (kastolars) [inline README.md]: Can we get a little more specific with what exactly to download? Perhaps just mention the zip folder and how to get the contents of it.

**Review by kastolars — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,58 @@
...
+Please select the branch that reflects your OS.
+[Linux Binary](https://github.com/SIGBlockchain/aurum_client/tree/linux_release)
+[Mac OS X Binary](https://github.com/SIGBlockchain/aurum_client/tree/mac_release)
+[Windows Binary](https://github.com/SIGBlockchain/aurum_client/tree/windows_release)
+Navigate to the bright green button on the right hand side that says clone or download. Click download zip.
+Find the zip file you downloaded and unzip to your desired directoy.
+Open a console or terminal window and navigate to the the aurum_client directory
+Type the following (if on a Linux / Mac OS X system - for Windows, please change the listed files to be executable and run):
+sudo chmod +x setup.sh
+sudo ./setup.sh
```
Comment (kastolars) [inline README.md]: I don't think sudo runs on Windows

File: README.md [hunk 5]
```diff
@@ -1 +1,58 @@
...
\ No newline at end of file
+<img src = "assets/aurum_logo_readme.jpg"  alt="drawing" width="200"/>
+
+## Installation
+Please select the branch that reflects your OS.
+[Linux Binary](https://github.com/SIGBlockchain/aurum_client/tree/linux_release)
+[Mac OS X Binary](https://github.com/SIGBlockchain/aurum_client/tree/mac_release)
+[Windows Binary](https://github.com/SIGBlockchain/aurum_client/tree/windows_release)
+Navigate to the bright green button on the right hand side that says clone or download. Click download zip.
+Find the zip file you downloaded and unzip to your desired directoy.
```
Comment (kastolars) [inline README.md]: For these binaries, can you simply add the link for the zip file?

File: README.md [hunk 6]
```diff
@@ -1 +1,58 @@
...
+  -setup
+    	set up client
+  -to string
+    	recipient
+  -update
+    	update wallet info
+  -version
+    	client version
+
+string means put the address of the person you want to interact with.
```
Comment (kastolars) [inline README.md]: What about for the `send` flag?

**Review by kastolars — CHANGES_REQUESTED** [review]

**Review by kastolars — APPROVED** [review]

**Pull request merged**